src/mbgl/map/transform.cpp

```diff
@@ -379,6 +379,16 @@ void Transform::setMaxZoom(const double maxZoom) {
     state.setMaxZoom(maxZoom);
 }
 
+void Transform::setMinPitch(const double minPitch) {
+    if (std::isnan(minPitch)) return;
+    state.setMinPitch(minPitch * util::DEG2RAD);
```

Comment:
Should we be checking against `util:: PITCH_MIN` here?

Reply:
Added warning